### DISCUSSION TITLE
Can anyone make a comparison of sixtyfps vs tauri?

### DISCUSSION THREAD

**Discussion by dclong** [slint-ui/slint#552] [General]
What are the advantages and disadvantages of sixtyfps compared to tauri?

**Comment by hunger** [comment]
This is based on way too little experience with Tauri and thus stays pretty abstract! Please correct me if I am wrong anywhere!

Both enable you to have UIs for rust programs.

Tauri leverages an entire browser engine to do so and enables you to write your UI using any of the many web frameworks available and ties that to rust code forming the backend. So Tauri combines Rust + Web Runtime + Web Frameworks into one package with a nice UI.

SixtyFPS uses its own language to describe UIs (inspired by web frameworks and looking somewhat similar to those), which is compiled into rust code and then combines that with more rust code you wrote to implement the applications functionality. No web engine, just rust code all the way!

Which one is better? I guess that depends a lot on your surrounding... when resource usage is not a critical concern and you have web frontend people around, then tauri is probably (I have not tried myself, so I am guessing;-) a great solution. If you are resource constrained or have only rust people, then sixtyFPS might be a better match for you.
